samples/deploy_and_test.py

```diff
@@ -819,14 +835,16 @@ def deploy(self):
     def delete(self):
         logging.info("Deleting '{}'".format(self.name))
         subprocess.run(['cf', 'delete', '-f', '-r', self.name], stdout=cf_logs)
+        subprocess.run(['cf', 'delete-orphaned-routes', '-f'], stdout=cf_logs)
         if self.app_router_name is not None:
             logging.info("Deleting '{}' app router".format(self.app_router_name))
             subprocess.run(['cf', 'delete', '-f', '-r', self.app_router_name], stdout=cf_logs)
         logging.info("Deleting '{}' Xsuaa service".format(self.xsuaa_service_name))
```

Comment:
this logging info should then go also in the if statement L844

Reply:
not sure, delete-orphaned-routes deletes routes from the application itself...